app/graphql/types/base_organization_type.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Types
+  class BaseOrganizationType < BaseObject
```

Comment:
Question: today organization types are located at the root of the `Types` module, mainly for legacy reasons.
Since almost all other types have their own sub-modules, should we use this PR as an opportunity to refact it?

Reply:
Agree with Vincent here. 
But as this PR fixes a production issue, can we perform the refactor after its merge?
Unless Julien already did it in the meantime, if that makes sense

Reply:
I will definitely move it to it's own subfolder!